docs/deployment_patterns/components/deployment_pattern_prerequisites.jsx

```diff
@@ -32,7 +32,7 @@ export default class Prerequisites extends React.Component {
 
   defaultPrerequisiteItems () {
     return [
-      <li key={0.1}>Completed the <a href='/docs/tutorials/getting_started/intro'>Getting Started Tutorial</a></li>
+      <li key={0.1}>Completed the <a href='docs/tutorials/getting_started/tutorial_overview'>Getting Started Tutorial</a></li>
```

Comment:
The link as it is currently won't direct to where you want it to --It will be treated as a relative link, so it'll end up pointing to `docs/deployment_patterns/components/docs/getting_started/tutorial_overview`.  Instead, you'll want to keep the leading / so that it is treated as an absolute path from the root, which should render as `/docs/tutorials/getting_started/tutorial_overview/`

```suggestion
      <li key={0.1}>Completed the <a href='/docs/tutorials/getting_started/tutorial_overview'>Getting Started Tutorial</a></li>
```